src/worker.cc

```diff
@@ -370,6 +370,12 @@ void parse_location(valhalla::Location* location,
   if (street_side_max_distance) {
     location->set_street_side_max_distance(*street_side_max_distance);
   }
+  auto street_side_cutoff =
+      rapidjson::get<std::string>(r_loc, "/street_side_cutoff", "service_other");
```

Comment:
why not optional? i worry that we set ourselves up to rely on this always being set. for example, in the pbf-only version of this api we do not parse json and so its totally possible to not set it.

Reply:
I made it optional now